pycbc/__init__.py

```diff
@@ -96,12 +95,9 @@ def sig_handler(signum, frame):
 
     signal.signal(signal.SIGUSR1, sig_handler)
 
-    if not verbose:
-        initial_level = logging.WARN
-    elif int(verbose) == 1:
-        initial_level = logging.INFO
-    else:
-        initial_level = int(verbose)
+    initial_level = logging.WARN
+    if verbose:
+        initial_level -= int(verbose) * 10
```

Comment:
Actually I think it could all be done in a single line:
`initial_level = logging.WARNING - 10 * int(verbose)`
since False maps to 30 - 0 = WARNING, True or 1 maps to INFO and 2 maps to DEBUG